modules/special-rules/pages/shooting-at-war-engines.adoc

```diff
@@ -29,9 +32,11 @@ Or, you may like to use coloured dice or 'damage tracker' dials -- when a War En
 
 == Allocation of hits in War Engine detachments
 
-Where the target detachment consists of more than one War Engine unit, allocate hits in the usual way.
-In other words, you must allocate hits evenly among the target War Engine units that are within the range and line of fire of your attacking units.
+Where the target detachment consists of more than one War Engine unit, allocate hits in the usual way. So allocate one hit to the nearest available target, then the next closest, and so on.
+// IJW editing note, I've tweaked this by copying text from S6 Allocate hits, to reinforce that you do the standard front-to-back hit allocation, one hit at a time.
+
 {blank}[TODO: Clarify whether we allocate against initial or current Damage Capacity. Current, I presume?]
+{blank}[TODO: reply IJW, as originally written, neither. You allocate as if they were regular units.]
```

Comment:
```suggestion
Where the target detachment consists of more than one War Engine unit, allocate hits in the usual way.
So, allocate one hit to the nearest available target unit, then the next nearest, and so on.
```

Note to self: We have a mix of 'closest' and 'nearest' at _S6 Allocate hits_, even in the same sentence. It's minor, but might as well update it to use a single term throughout the topic, and to match the edit I've made in this suggestion (that is, use 'nearest').

Reply:
Aye and aye.